lib/puppet/agent/locker.rb

```diff
@@ -29,7 +29,7 @@ def lock
 
   # @deprecated
   def running?
-    Puppet.deprecation_warning _(<<-ENDHEREDOC)
+    Puppet.deprecation_warning <<-ENDHEREDOC
```

Comment:
Apparently this is how we externalize heredoc comments. I'm going to put up a PR to restore those.

Reply:
https://github.com/puppetlabs/puppet/pull/6236

Reply:
Yes, definitely. I really doubt those heredocs are going to cause errors. Nice catch on those other standalone words though.